yarn.lock

```diff
@@ -257,6 +236,7 @@ csv-parse@^4.6.5:
   resolved "https://registry.yarnpkg.com/csv-parse/-/csv-parse-4.16.3.tgz#7ca624d517212ebc520a36873c3478fa66efbaf7"
   integrity sha512-cO1I/zmz4w2dcKHVvpCr7JVRu8/FymG5OEpmvsZYlccYolPBLoVGKUHgNoc4ZGkFeFlWGEDmMyBM+TTqRdW/wg==
 
+<<<<<<< Updated upstream
```

Comment:
@cdoyle-temple There are some merge markers here that need to be fixed.

Reply:
Yes, I can see them now